docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -79,11 +80,41 @@ Here are the steps to configure a new Google Service Account Key in EAS for send
 </Step>
 
 <Step label="5">
+  Configure the `google-services.json` file in your app. (Note: you may skip this step if `google-services.json` has already been set up.)
+
+  First, download the `google-services.json` file from Firebase Cloud Console and place it somewhere in your app directory (the root is fine). 
+
+  <ImageSpotlight
+    alt="Download google-services.json from Firebase Cloud Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/05-setup-google-services-json.png"
+    style={{ maxWidth: 800 }}
+  />
+
+  Then update your app.json with the key `expo.android.googleServicesFile` and value `path/to/google-services.json`.
+
+  <CodeBlocksTable>
+    ```json
+	// app.json
+	{
+	  "expo": {
+		...
+		"android": {
+		  ...
+		  "googleServicesFile": "./path/to/google-services.json"
+		},
+	  }
+	}
+    ```
+  </CodeBlocksTable>
```

Comment:
````suggestion
    ```json app.json

    {
      "expo": {
    	/* @hide ...*/ /* @end */
    	"android": {
    	  /* @hide ...*/ /* @end */
    	  "googleServicesFile": "./path/to/google-services.json"
    	},
      }
    }
    ```
````

We don't have to use CodeBlocksTable here unless there is a side-by-side comparison of two code snippets  Currently, this is how it renders:
![CleanShot 2024-03-14 at 17 58 38](https://github.com/expo/expo/assets/10234615/1f521393-9337-4656-a7f6-3bf4d36127ee)

- Let's remove the component and use the code snippet display pattern as we usually do in our docs (such as instead of using three dots, we use `@hide` annotation and the file name can be with the snippet language code)